WindowsDevicePortalWrapper/WindowsDevicePortalWrapper.Shared/IoT/ApplicationManager.cs

```diff
@@ -109,13 +113,13 @@ public class AppsListInfo
             /// Gets or sets the default application 
```

Comment:
The comments need to be updated now to say "Gets ..." instead of "Gets or sets ..."